src/Language/PureScript/Ide/Types.hs

```diff
@@ -149,16 +149,16 @@ instance ToJSON ModuleImport where
     object $ [ "module" .= mn
              , "importType" .= ("implicit" :: Text)
              ] ++ fmap (\x -> "qualifier" .= x) (maybeToList qualifier)
-  toJSON (ModuleImport mn (P.Explicit refs) _) =
-    object [ "module" .= mn
+  toJSON (ModuleImport mn (P.Explicit refs) qualifier) =
+    object $ [ "module" .= mn
            , "importType" .= ("explicit" :: Text)
            , "identifiers" .= (identifierFromDeclarationRef <$> refs)
-           ]
-  toJSON (ModuleImport mn (P.Hiding refs) _) =
-    object [ "module" .= mn
+           ] ++ fmap (\x -> "qualifier" .= x) (maybeToList qualifier)
```

Comment:
Hmm, this was ambiguous due to the non-qualified import of `Data.Map.Lazy`. Had no idea Haskell had both `import qualified ... as ...` and `import ... as ...` forms...